src/test/config/jira/helm_parameters

```diff
@@ -11,4 +12,8 @@ TARGET_NAMESPACE=${kubernetes.target.namespace}
 DOCKER_IMAGE_REGISTRY=${dockerImage.registry}
 DOCKER_IMAGE_VERSION=${dockerImage.version}
 INGRESS_DOMAIN_KITT=${kitt.ingress.domain}
-INGRESS_DOMAIN_EKS=${eks.ingress.domain}
\ No newline at end of file
+INGRESS_DOMAIN_EKS=${eks.ingress.domain}
+# .Values.replicaCount=1, but in here we specify how many nodes we are planning to get to prepare ingress rules for each
+TARGET_REPLICA_COUNT=2
```

Comment:
Can we just hard-code this for now? We're unlikely to go beyond 2 nodes in testing.

Reply:
Hmm, I'm allergic to hardcoding. Why do you think it's a problem in this case? It would allow us to change a for loop with something flat, but I'm not sure if that's much of a gain.